homeassistant/components/olarm/strings.json

```
@@ -0,0 +1,39 @@
+{
+  "config": {
+    "step": {
+      "device": {
+        "title": "Select device",
+        "description": "Choose the Olarm device you would like to integrate. If your device is not listed, please ensure you have enabled public API access in your Olarm account for that device.",
+        "data": {
+          "select_device": "Device",
+          "load_zones_bypass": "Load entities for zone bypassing?"
+        },
+        "data_description": {
+          "select_device": "",
```

To not leave this empty, how about:

```suggestion
          "select_device": "The Olarm device you would like to integrate",
```